osu.Game.Rulesets.Mania/Mods/ManiaMod.cs

```diff
@@ -146,6 +149,7 @@ public class ManiaModKey9 : ManiaKeyMod
     public class ManiaModKeyCoop : Mod
     {
         public override string Name => "KeyCoop";
+        public override string ShortenedName => "CO";

```

Comment:
Where did you get these shortened names from? Are these coming from osu!web? Because I know that at least KeyCoop is wrong (this should be `2P`).